src/evict/evict_lru.c

```diff
@@ -2053,15 +2053,15 @@ __evict_walk_tree(WT_SESSION_IMPL *session, WT_EVICT_QUEUE *queue, u_int max_ent
         if (F_ISSET(ref, WT_REF_FLAG_INTERNAL))
             internal_pages_queued++;
 
-        __wt_verbose(session, WT_VERB_EVICTSERVER, "select: %p, size %" WT_SIZET_FMT, (void *)page,
-          page->memory_footprint);
+        __wt_verbose_debug3(session, WT_VERB_EVICTSERVER, "select: %p, size %" WT_SIZET_FMT,
```

Comment:
Selecting a page means quite important would revert this line to original importance

Reply:
Thanks, fixed in ce6c52b.